tests/TreeHouse/WorkerBundle/QueueManagerTest.php

```diff
@@ -884,6 +884,29 @@ public function testFatalErrorCatch()
 
         $this->assertEquals(false, $this->manager->executeJob($job));
     }
+
+    public function testFatalErrorCatchWithNoRetries()
```

Comment:
I'm not sure what this test adds in comparison to the previous test. I can see the difference but what fix does this test, and is it part of this PR?

Reply:
Thank you for your feeback - yes the difference between those two tests is subtle.

My test sets the `releases` attribute to 2 so the `JobBuriedEvent` is created. The previous test only reschedules the job and does not create the `JobBuriedEvent`. Maybe I can add an assertion that the `JobBuriedEvent` is dispatched to make this more explicit.

Reply:
I have now updated the testcase to check if the buried event has actually been dispatched. I hope the intent of the test is now clearer @pkruithof